src/rename_samples.py

```diff
@@ -0,0 +1,22 @@
+import click
+
+from wrappers.cobs import Cobs
+
+
+@click.command()
+@click.argument('infile', type=click.File('r'))
+def rename_samples(infile):
+    mapping = {}
+
+    lines = infile.readlines()
+    for line in lines:
+        line = line.strip()
+        old, new = line.split('\t')
+        mapping[old] = new
+
+    cobs = Cobs()
```

Comment:
Can we add the classic index directory as an argument instead of having to set an environment variable to run it?